packages/cli/src/lib/packager/index.ts

```diff
@@ -67,6 +67,11 @@ type Options = {
    */
   buildDependencies?: boolean;
 
+  /**
+   * When `buildDependencies` is set, this list of packages will not be build even if they are dependencies.
+   */
+  buildExcludes?: string[];
```

Comment:
Hm why was this one added?

Reply:
So that we can exclude the target package, e.g. `packages/backend`. It's built before and separately since the idea is that this will be used as the single build command in the backend package. Running its own build script might cause a loop.

Reply:
I see. Ok, I guess I wasn't sure why it wasn't needed before.